### PR TITLE
Devops 37: Write a liquibase update task and apply it to the test environment upon deployment

### PR DESCRIPTION
**Pull request by NateHand93**
So, it turns out I was a little too focused on how to get Liquibase to play nice with gradle. What I actually should have been focusing on is getting Liquibase to play nice with Spring Boot, which is a much much _much_ _much_ _**much**_ simpler task to implement. If Spring boot detects Liquibase on the classpath, on startup, the application will automatically run the changesets in the master changelog specified by the `liquibase.change-log` property.

With that being said, the implementation was pretty straightforward. I consolidated all changesets to one master changelog, moved the changelog to a location that automatically gets added to the classpath (and hence, the jar) by gradle, and added the `liquibase.change-log` property to `startup-wrapper.sh`.

This has only been tested locally thus far, but because we have our datasource already configured in the test environment with root credentials, Liquibase should be able to run without any hiccups.

### CODE REVIEW

**Comment by RussianPrussian** [comment]
Okay - so what's our roll back plan if we merge this and we end up with some sort of DB mismatch or something. Do we need to completely disassemble the test environment database first? Do we need some sort of condition in the scripts : "i.e. if table doesn't exist, then create it"

File: devops-accounts-service/application-localdev.properties [hunk 1]
```diff
@@ -2,6 +2,8 @@ server.port=8080
 
 #Local DB Creds
 spring.datasource.url=jdbc:mysql://33.33.33.1/psiappdata
-spring.datasource.username=psidatauser
-spring.datasource.password=H8[kXC7GzEFkC5kt
-security.basic.enabled=false
\ No newline at end of file
+spring.datasource.username=root
+spring.datasource.password=root
```
Comment (RussianPrussian) [inline devops-accounts-service/application-localdev.properties]: do we need to log in as root to make this work?

Comment (RussianPrussian) [reply devops-accounts-service/application-localdev.properties]: and is spring boot going to run the whole thing? or an update?

**Comment by NateHand93** [comment]
@RussianPrussian So, included in the changelog is a bit of SQL that essentially does disassemble the test database - all tables will be dropped. Every changeset has an id associated with it, and once the changeset is successfully executed, that id will be added to the `DATABASECHANGELOG` table (which Liquibase automatically creates). Once the id of the changeset is added to to `DATABASECHANGELOG`, it won't be executed again. So, essentially, Liquibase is doing the "if the table doesn't exist, then create it" for us.

File: devops-accounts-service/application-localdev.properties (hunk 1 above)
Comment (NateHand93) [reply devops-accounts-service/application-localdev.properties]: We don't _need_ to log in as root. It is just convenient that the root user has privileges to run all of the commands in the changelog. Spring boot will run any changeset that hasn't already been executed on the database, in the order that they appear in the changelog file.

**Comment by RussianPrussian** [comment]
Ah ha. Clever. I'll take that.

File: devops-accounts-service/application-localdev.properties (hunk 1 above)
Comment (RussianPrussian) [reply devops-accounts-service/application-localdev.properties]: Okay. This is fine for now, but let's look at changing it. Also - does the RDS user have the appropriate privileges? Is that user the root one?

Comment (NateHand93) [reply devops-accounts-service/application-localdev.properties]: Yep, the RDS user has the appropriate privileges

**Pull request merged**